e2e-tests/graphql-spring-examples/pom.xml

```diff
@@ -0,0 +1,151 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.0.2.RELEASE</version>
+        <relativePath/>
+    </parent>
+
+
+    <artifactId>evomaster-e2e-tests-graphql-spring-examples</artifactId>
+    <packaging>jar</packaging>
+
+
+
+    <dependencies>
+
+        <!--
+            Issue with Jersey (coming from client-java) using old version
+        -->
+        <dependency>
```

Comment:
dependency versions should not be here, but rather in the root pom file.
if you do not specify \<version\>, then it is inherited from \<parent\>